app/controllers/books/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Books::CommentsController < CommentsController
+  before_action :set_commentable
+  before_action :set_rendar
```

Comment:
```suggestion
  before_action :set_render
```

綴りが間違っていそうです〜

あと、このメソッド名だと set しているのは template だと思うので、それとわかるメソッド名にしたいです！（というか `render` は名詞でなく動詞なので文法的に意味が通らないと思います。[参考](https://eow.alc.co.jp/search?q=render)）

Reply:
おお！rendarは動詞ですね...set_templateに変更いたしました！